Source/hu_stuff.c

```diff
@@ -1144,7 +1144,8 @@ void HU_Drawer(void)
       }
 
       // [FG] draw player coords widget
-      if (automapactive && !(hud_distributed && automapoverlay) && map_player_coords)
+      if ((automapactive && STRICTMODE(map_player_coords) == 1)
```

Comment:
How about the `hud_distributed` condition?

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/597/commits/0b742ae86ed429ac0fbf683e73edf0b8a25bab2b